dpppt-config-backend/src/main/java/org/dpppt/switzerland/backend/sdk/config/ws/controller/DPPPTConfigController.java

```diff
@@ -32,31 +33,42 @@
 @Controller
 @RequestMapping("/v1")
 public class DPPPTConfigController {
-	
+
 	private static final String IOS_VERSION_DE_WEEKLY_NOTIFCATION_INFO = "ios13.6";
-	private static final List<String> TESTFLIGHT_VERSIONS = List.of("ios-200619.2333.175", 
+	private static final List<String> TESTFLIGHT_VERSIONS = List.of("ios-200619.2333.175",
 			   "ios-200612.2347.141",
 			   "ios-200528.2230.100",
 			   "ios-200524.1316.87",
 			   "ios-200521.2320.79");
 
-	
+
 	private static final Logger logger = LoggerFactory.getLogger(DPPPTConfigController.class);
 
 
 	public DPPPTConfigController() {
 	}
 
+	@Documentation(description = "Echo endpoint",
+			responses = {"200 => Hello from DP3T Config WS"})
 	@CrossOrigin(origins = { "https://editor.swagger.io" })
 	@GetMapping(value = "")
 	public @ResponseBody String hello() {
 		return "Hello from DP3T Config WS";
 	}
 
+	@Documentation(description="Read latest configuration and messages, depending on the version of the phone and the" +
+			" app.",
+			responses = {"200 => ConfigResponse structure with eventual notifications and epidemic parameters"})
 	@CrossOrigin(origins = { "https://editor.swagger.io" })
 	@GetMapping(value = "/config")
-	public @ResponseBody ResponseEntity<ConfigResponse> getConfig(@RequestParam(required = true) String appversion,
-			@RequestParam(required = true) String osversion, @RequestParam(required = true) String buildnr) {
+	public @ResponseBody ResponseEntity<ConfigResponse> getConfig(
+			@Documentation(description = "Version of the OS", example = "ios13.6")
```

Comment:
now there are two `@Documentation` annotation on the same Parameter :octocat:

Reply:
Ah, this is where the failure comes from - should be fixed in the latest update...